### ISSUE TITLE
HmIP-BROLL: Rollo-Statusmeldung immer falsch

### ISSUE THREAD

**Issue by s0n1c** [thkl/homebridge-homematic#564]
Der übertragene Status von HmIP-BROLL Devices ist immer "Beim Schließen...", egal ob die Dinger geöffnet, geschlossen oder in Bewegung sind. 

Bei HM-LC-Bl1PBU-FM gibt es dieses Problem nicht. Wurde eventuell ein Datenpunkt falsch hinterlegt?

Zum Nachstellen: Homematic (letzte Version) mit HmIP-BROLL und Homebridge (letzte Version) samt homebridge-homeatic (letzte Version) nutzen und schon hat man den Salat.

**Comment by thkl** [comment]
Ok muss ich irgendwie simulieren. Ich habe den Aktor nicht.

**Comment by hillinator** [comment]
Hi, ich kann das Phänomen bestätigen. Unabhängig vom tatsächlichen Status, wird immer "Beim Schließen" angezeigt. Obwohl Alle Jalousien Offen sind, werde diese aber auch mal geschlossen und mal offen angezeigt. In der CCU3 wird der Status korrekt angezeigt.

@thkl würde ein Debug Log weiterhelfen?

![image](https://user-images.githubusercontent.com/60514784/78112096-fefaf080-73fd-11ea-9c2f-9e9e1177e603.png)

**Comment by abcklaus** [comment]
Ich verwende die HmIP-FROLLs. Diese zeigen ebenfalls oben beschriebenes Problem.

**Comment by ben2o** [comment]
Hab ein FROLL verbaut und bei mir wird Öffnen statt offen angezeigt.
geschlossen wird richtig angezeigt.

**Comment by thkl** [comment]
Ja das kenne ich auch aktuell aus dem hap-homematic Projekt. Die Englische Version der Home App zeigt komischerweise Close -> Opening -> Open an ... die deutsche Geschlossen -> Öffnen

**Comment by abcklaus** [comment]
Stimmt, hab es grad nachgestellt und um mein iPhone auf "englisch" umgestellt. Im englischen passen die Begriffe. Es scheint also ein Übersetzungsproblem zu sein. 

Im Englischen ist mir dann noch aufgefallen, dass der Status für "opening" und "closing" vertauscht ist. Beim Runterfahren der Rolladen wird "opening" angezeigt und vice versa. Kann das jemand bestätigen?

In der Homematic Weboberfläche werden die Status (auch für öffnend und schliessend) korrekt angezeigt. Dementsprechend schliess ich eine Vertauschung im Autor aus.

**Comment by mainkowitsch** [comment]
ich habe da noch ein problem was möglicherweise damit zusammenhängt.
Wenn ich über einen Taster (HmIP-FCI6) eine jalousie halb schließe  funktioniert die automatische schließung über eine Automation in Homekit genau an diesem fenster nicht mehr, da mir Hombridge "Geschlossen" anzeigt. In der CCU wird mir allerdings der richtige % Wert ausgegeben.

Jalousie oben -> taster runter drücken bis gewünschte höhe -> taster hoch drücken zum stoppen (gewünschte höhe erreicht)
CCU > richtiger % Wert
Homebridge > geschlossen

**Comment by mainkowitsch** [comment]
ich habe da noch ein problem was möglicherweise damit zusammenhängt.
Wenn ich über einen Taster (HmIP-FCI6) eine jalousie halb schließe funktioniert die automatische schließung über eine Automation in Homekit genau an diesem fenster nicht mehr, da mir Hombridge "Geschlossen" anzeigt. In der CCU wird mir allerdings der richtige % Wert ausgegeben.

Jalousie oben -> taster runter drücken bis gewünschte höhe -> taster hoch drücken zum stoppen (gewünschte höhe erreicht)
CCU > richtiger % Wert
Homebridge > geschlossen

**Comment by SmartScrimper** [comment]
> Ok muss ich irgendwie simulieren. Ich habe den Aktor nicht.

Kann man evtl. helfen? Hab das Problem auch.
